CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Add rule GCI96 Avoid CSV Format, this rule is designed for Python but it can be implemented in other languages. It's based on the utilization of the CSV format, which is not considered an efficient format for data storage and transfer. The rule suggests using more efficient formats like Feather or Parquet instead of CSV.
```

Comment:
GCI96 is already used by another rule ... change to GCI99 please. make the changes to GCI99 in all your PR.